src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSFunctionBinding.cs

```diff
@@ -128,6 +136,11 @@ public static void InvokeJS(JSFunctionBinding signature, Span<JSMarshalerArgumen
         /// Locates and binds a JavaScript function given name and module so that it can later be invoked by managed callers.
         /// This API supports JSImport infrastructure and is not intended to be used directly from your code.
         /// </summary>
+        /// <param name="functionName">Name of the exported JavaScript function.</param>
+        /// <param name="moduleName">Name of the ES6 module</param>
+        /// <param name="signatures">Metadata about the signature of the marshaled parameters.</param>
+        /// <returns>Method metadata.</returns>
+        /// <exception cref="PlatformNotSupportedException">Will be thrown when the method was execute on architecture other than WebAssembly.</exception>
```

Comment:
Exception messages should start as if they were implicitly preceded by "The exception that is thrown when...".

```suggestion
        /// <exception cref="PlatformNotSupportedException">The method is executed on an architecture other than WebAssembly.</exception>
```